spec.md

```diff
@@ -236,14 +246,17 @@ within the same JSON object.
 ### id
 
 - Type: `String`
-- Description: ID of the event. The semantics of this string are explicitly
-  undefined to ease the implementation of producers. Enables deduplication.
+- Description: Identifies the event for de-duplication.
+  Producers MUST ensure that each distinct event from the same `source` has a unique `id`  If a duplicate event is re-sent due to a network error, it MAY have the same `id`.
+.
+  Consumers MAY assume that Events with identical `source` and `id` are duplicates.
 - Examples:
+  - An event counter maintained by the producer
   - A database commit ID
```

Comment:
Can we remove this example (DB commit ID)? It's a bad example and will lead to confusion

Reply:
done